### PR TITLE
Change testing to dummy nanoaod file and add docs how to generate it

### PR DESCRIPTION
**Pull request by stwunsch**

### CODE REVIEW

File: docs/sphinx_source/create_nanoaod.rst [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+==========================================
+
+We use self-generated NanoAOD samples for testing, which can be produced with the open source [CMSSW](https://github.com/cms-sw/cmssw) framework. The following script generates a file from scratch using [fast simulation](https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideFastSimulationExamples).
+
+The used files are hosted in the repository [CROWNTestingSamples](https://github.com/KIT-CMS/CROWNTestingSamples).
+
+It should be noted that these files must be used solely for testing purposes and do not contain any useful information for physics analysis.
+
+.. code-block:: console
+    #!/bin/bash
```
Comment (harrypuuter) [inline docs/sphinx_source/create_nanoaod.rst:11]: need an empty line here (due to sphinx)

**Review by harrypuuter — CHANGES_REQUESTED** [review]

File: docs/sphinx_source/create_nanoaod.rst (hunk 1 above)
Comment (stwunsch) [reply docs/sphinx_source/create_nanoaod.rst:11]:
```suggestion

    #!/bin/bash
```

Like that?

Comment (harrypuuter) [reply docs/sphinx_source/create_nanoaod.rst:11]: yes

**Comment by stwunsch** [comment]
@harrypuuter it's in!

**Review by harrypuuter — APPROVED** [review]

**Pull request merged**